module_3/task3.js

```diff
@@ -0,0 +1,42 @@
+import csvtojson from "csvtojson"
+import fs from "fs"
+const CSV_FILE_PATH = ".csv/data-csv.csv"
+
+async function fromCSVtoTXT() {
+  try {
+    const onError = (err) => {
+      console.error(err)
+    }
+
+    const onComplete = () => {
+      console.log("completed")
+    }
+
+    csvtojson()
```

Comment:
the TXT file is containing the `amount` property which is not expected, you should ignore it when writting to TXT file